janus_core/helpers/post_process.py

```diff
@@ -212,6 +213,9 @@ def compute_vaf(
     filter_atoms : MaybeSequence[MaybeSequence[Optional[int]]]
         Compute the VAF averaged over subsets of the system.
         Default is all atoms.
```

Comment:
This isn't newly introduced, but to me this isn't entirely clear how this is used (why could be be a sequence of a sequence?)

I also wonder if 

```
    filter_atoms = [
        atom if atom[0] is not None else range(n_atoms) for atom in filter_atoms
    ]
```

etc. is as clear as it could be that `atom` is generally referring to an index, rather than `Atom` objects as if often the case for similar code elsewhere.

Reply:
On ```MaybeSequence[MaybeSequence[Optional[int]]]``` I think the idea is to be able to do 

```python
   vaf_post = post_process.compute_vaf(
        traj, use_velocities=True, filter_atoms=(na, cl)
    )
```

where ```(na, cl) === ( (0, 2, 4, ...), (1, 3, 5, ...) )```, which is a sequence of sequence. That way you get the partial VAF's for Na and Cl at once.

The one liner could use some thought, I wasn't sure what to do last pr.

Incidentally I have tested rebasing #285 onto this and all is fine (just will need to account for the new lags in the return type).